res/style.css

```diff
@@ -57,8 +58,13 @@ ul.navbar li {
     vertical-align: middle;
     text-align:     center;
     display:        inline-block;
-    line-height:    normal;
+    line-height:    48px;
```

Comment:
Yes, I know... I am asking if it works well on all **window** dimensions and text lengths(e.g. multiline).